Ajout des textes des campagnes non importées par l'éditeur (#91)
nomises/Traduction-Hota-FR: Traduction/campagne_tempo_txt/H5AllIn/valeurNonImporte.md

### Epilogue

## 0503
### Regions right-click
Dargem apprend que ses employeurs ont l'intention d'organiser une expédition à la recherche de la légendaire Corne des Abysses, et s'introduit illégalement dans AvLee pour y prélever du bois et construire une flotte en vue de cette grande entreprise. 	

### Prologue
Je m’étais habitué à cette sensation déchirante de n’avoir rien au-dessus de ma tête — qu’il s’agisse de la voûte d’anciens tunnels, d’un toit solide ou, à tout le moins, d’un plafond de pont qui tangue. Rares sont ceux qui, à Nighon, peuvent supporter cela, mais je m’y suis adapté ! Seuls les bois m'inspirent encore une terreur honteuse et peu virile. Bien sûr, personne ne me le fera jamais remarquer — après tout, garder son sang-froid est la marque d’un maître dans ma profession. Ce journal est le seul confident à qui je livre mes véritables émotions. Je pense que je préférerais de loin être enterré vivant une fois de plus… Mais le contrat est signé, ce qui signifie que Dargem Llywellyn se dirige vers les fourrés maudits. Et par tous les démons de la mer, il reviendra victorieux.

### Epilogue

## 0504
### Regions right-click

> Dargem se rend au cœur de Deyja, le pays des Nécromanciens, afin de fouiller la bibliothèque d'Archibald Ironfist à la recherche d'indices sur l'emplacement possible de la Corne des Abysses.

Reviewer: ```suggestion
Dargem se rend au cœur de Deyja, le pays des nécromanciens, pour piller la bibliothèque d'Archibald Poing-de-Fer à la recherche d'indices sur l'emplacement possible de la Corne des Abysses.
```

### Prologue

> J'ai vu bon nombre de merveilles, et je ne suis pas facilement impressionné… mais la vue de cette bibliothèque monumentale m'a rempli d'admiration. On raconte qu'il y a mille ans, lorsque les Nécromanciens furent bannis de Bracada et s'installèrent à Deyja, elle s'élevait déjà ici, ses voûtes renfermant d'innombrables secrets. Aucune des modifications architecturales récentes, plutôt de mauvais goût — manifestement l'œuvre de ces amateurs de charognes — ne pouvait éclipser la véritable grandeur des Anciens, toujours visible dans ses formes. Eh bien, c'est un grand honneur de pouvoir toucher quelque chose d'intemporel. Mieux encore, d'en emporter un morceau. En souvenir.

Reviewer: ```suggestion
J'ai vu bien des merveilles, et je ne suis pas facilement impressionné... mais la vue de cette bibliothèque monumentale m'a saisi d'admiration. On raconte qu'il y a mille ans, lorsque les nécromanciens furent bannis de Bracada et s'installèrent à Deyja, elle se dressait déjà ici, ses voûtes recelant d'innombrables secrets. Aucune des modifications plus récentes, plutôt de mauvais goût, apportées à son architecture - manifestement la marque de ces amateurs de charogne - ne saurait éclipser la véritable grandeur des Anciens, encore perceptible dans ses formes. Quel honneur de pouvoir toucher quelque chose d'intemporel ! Mieux encore, d'en emporter un fragment. En souvenir.
```

### Epilogue

## 0505
### Regions right-click
La route de Dargem le mène vers un archipel situé près de Tatalia ; là-bas, dans les grottes situées sous l'Île du Serpent Vengeur, il part à la recherche de la Corne des Abysses, une relique que tout pirate rêve de posséder. 

### Prologue

> C'est parfois comme ça : on sait très bien qu'on se fait mener en bateau, et pourtant… on apprécie le moment. Ma rencontre avec cette employeuse énigmatique était à la fois déprimante et exaltante. J'ai tout de suite su que cette femme magnifique était complètement folle — personne de sensé ne ferait autant de bruit autour d'un simple artéfact, quels que soient les pouvoirs qu'il était censé détenir. Et pourtant, je me suis retrouvé de très bonne humeur, pour la même raison : cette Corne des Abysses pouvait sûrement rapporter gros. Le reste ne devrait être qu'un petit défi : la trouver... et régler la délicate question des droits sur cette découverte.

Reviewer: ```suggestion
Il arrive parfois que l'on sache pertinemment qu'on se joue de nous, et pourtant... on y prend plaisir. Ma rencontre avec cette employeuse énigmatique fut à la fois déprimante et exaltante. J'ai tout de suite compris que cette femme magnifique était complètement folle - personne de sensé ne ferait tout un plat d'un simple artefact, quels que soient les pouvoirs qu'on lui prêtait. Et pourtant, je me suis retrouvé de bonne humeur, pour la même raison : cette Corne des Abysses pourrait sans aucun doute rapporter gros. Le reste ne devrait être qu'une formalité : la trouver... et régler la question délicate des droits de propriété.
```
